Documentation/dev-guide/api_reference_v3.md

```diff
@@ -616,6 +616,7 @@ Empty field.
 | value | value is the value, in bytes, to associate with the key in the key-value store. | bytes |
 | lease | lease is the lease ID to associate with the key in the key-value store. A lease value of 0 indicates no lease. | int64 |
 | prev_kv | If prev_kv is set, etcd gets the previous key-value pair before changing it. The previous key-value pair will be returned in the put response. | bool |
+| preserve_value | If preserve_value is set, etcd overwrites the key with previous value. Use this to detach a lease by overwriting the key. | bool |
```

Comment:
`If preserve_value is set, etcd preserves the key by detaching lease if any associated with it. Use this to detach the lease from the key.` instead?